src/cloud/project/services-elastic.md

```diff
@@ -199,5 +199,26 @@ elasticsearch:
 If you use the ElasticSuite third-party plugin, you must [update the `{{site.data.var.ct}}` package]({{ site.baseurl }}/cloud/project/ece-tools-update.html) to version 2002.0.19 or later.
 When setting up ElasticSuite, add the configuration settings to the `ELASTICSUITE_CONFIGURATION` deploy variable. This configuration saves the settings across deployments.
 
+### Remove plugins for Elasticsearch
+Removing the plugin entries from `elasticsearch:` in `.magento/services.yaml` does not uninstall or disable them as you might expect. You must take the additional step of reindexing your Elasticsearch data. This is intentional to prevent possible loss or corruption of data that depends on these plugins.
+
+**To remove Elasticsearch plugins:**
+
+1. Remove the Elasticsearch plugin entries from your `.magento/services.yaml` file.
+
+2. Reindex the Catalog Search index:
+
+    ```bash
+    bin/magento indexer:reindex catalogsearch_fulltext
+    ```
+
+3. Clean the cache:
+
+    ```bash
+    bin/magento cache:clean
+    ```
+
+4. Commit the `.magento/services.yaml` changes to your Cloud repo.
```

Comment:
Based on your explanation - surely this step should be done BEFORE reindexing? IE remove the plugins and then reindex. Otherwise just editing the YAML file won't achieve anything? (Except locally)

Reply:
Yeah, based on the info I have so far, this is intended to be a last step so devs can do a local check to ensure the reindexing removed the plugins affects. I'm waiting for @andriyShevtsov to provide a bit more clarity for this PR.

Reply:
Just saw the thumbs up. Cool.